### ISSUE TITLE
Decrypting ciphertext results in "-nan"

### ISSUE THREAD

**Issue by duanglukui** [eth-sri/phoenix#2]
I am currently facing an issue while running Phoenix code, where the decryption of ciphertext always results in "-nan" .   I am eagerly looking forward to your response and guidance to help resolve this issue.

The incorrect running result is as follows:

------------------------------>Collecting batch 5/5
Homomorphic inference:
printing first 32 logit arrays
logits: [-nan -nan -nan -nan -nan | -nan -nan -nan -nan -nan -nan -nan -nan -nan -nan -nan], pred: 0 (WRONG)
logits: [-nan -nan -nan -nan -nan | -nan -nan -nan -nan -nan -nan -nan -nan -nan -nan -nan]
logits: [-nan -nan -nan -nan -nan | -nan -nan -nan -nan -nan -nan -nan -nan -nan -nan -nan]

Plain inference:
printing first 32 logit arrays
logits: [0.0135033 0.012327 0.0145569 0.0138069 0.0104689 | 0.0121103 0.0120496 0.0114161 0.0128309 0.0110686 0 0 0 0 0 0] , pred: 2 (OK)

**Comment by chenyusii** [comment]
i also meet this problem.

**Comment by duanglukui** [comment]
@chenyusii  Hi! Since we are facing the same issue, feel free to email me at 2110436077@email.szu.edu.cn if you'd like to discuss more details.

**Comment by rand0musername** [comment]
Hi all, sorry for the huge delay in writing back, and thank you for your interest. I unfortunately do not have bandwidth to look into this more deeply anymore but seems quite unusual and is definitely not something we observed with the final version of the code. If you found a solution in the email discussion feel free to share and help those who encounter this in the future.
